MovieAppPetProject/MovieAppPetProject/Network/Constants.swift

```diff
@@ -17,4 +17,13 @@ struct Constants {
         static let upcomingMoviesURL = "https://api.themoviedb.org/3/movie/upcoming?api_key=\(apiKey)"
         static let searchMoviesURL = "https://api.themoviedb.org/3/search/movie?query=fight%20club&include_adult=false&api_key=\(apiKey)"
     }
+
+    struct Identifiers {
+        static let goToMovieDetails = "GoToMovieDetails"
+        static let homeScreenCollectionViewCell = "HomeScreenCollectionViewCell"
+        static let searchScreenTableViewCellNibIdentifier = "SearchMovieTableViewCell"
+        static let homeScreenTableCell = "HomeScreenTableCell"
+        static let searchScreenTableCell = "SearchScreenTableCell"
+        static let moviePosterPath = "https://image.tmdb.org/t/p/w500"
```

Comment:
This is not an identifier. Maybe create another class that has your paths

Reply:
ah damn hadn't noticed I had it in there, will do